website/src/components/Changelog/GUI.tsx

```diff
@@ -27,10 +27,26 @@ export default function GUI({ title }: { title: string }) {
         </ul>
       </Entry>
       */}
+      <Entry version="1.1.12" date={new Date("2024-08-13")}>
+        <ul className="list-disc space-y-2 pl-4 mb-4">
+          <ChangeItem pull="6226">
+            Fixes a bug where clearing the log files would delete the current
+            logfile, preventing logs from being written.
+          </ChangeItem>
```

Comment:
Remove #4764 from the list of known issues for the Linux and Windows GUI Clients in `client-apps`, since this PR closed that issue

Reply:
nvm got it

Reply:
thanks!